frontend/packages/kubevirt-plugin/src/components/form/popover-helper.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { Popover, PopoverPosition } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+
+type HelpIconPopoverProps = {
+  bodyContent: React.ReactNode;
```

Comment:
we could pass the bodyContent as children

Reply:
this could be misleading. are we sure about this?

Reply:
whatever you prefer. I usually like passing as children as its more readable to me.